pkg/peertls/templates.go

```diff
@@ -20,6 +21,7 @@ func CATemplate() (*x509.Certificate, error) {
 		ExtKeyUsage:           []x509.ExtKeyUsage{x509.ExtKeyUsageServerAuth},
 		BasicConstraintsValid: true,
 		IsCA:                  true,
+		Subject:               pkix.Name{CommonName: "Storj"},
```

Comment:
Subject CN is pretty much deprecated, I think. https://security.stackexchange.com/a/69160

Reply:
Ok, it looks that I need one from three (C,O,CN) to be able to use this cert with minio and Filezilla PRO. I will use 'Organization'. Thanks